Geometry/HGCalCommonData/src/HGCalWaferMask.cc

```diff
@@ -333,6 +333,42 @@ bool HGCalWaferMask::goodCell(int u, int v, int n, int type, int rotn) {
   return good;
 }
 
+int HGCalWaferMask::getRotation(int zside, int type, int rotn) {
+  int newrotn(rotn);
+  if ((zside < 0) && (type != HGCalTypes::WaferFull)) {
```

Comment:
should you check if `rotn` is less than `6` for these cases?
(could also make `6` a shared, named constant)

Reply:
Done and committed